test/regression/targets/sc-60301.cc

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file   sc-60301.cc
+ *
+ * @section LICENSE
+ *
+ * The MIT License
+ *
+ * @copyright Copyright (c) 2024 TileDB, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#include <test/support/tdb_catch.h>
+#include <tiledb/tiledb>
+
+using namespace tiledb;
+
+static void create_array(Context& ctx, const std::string& array_uri);
+static void write_array(Context& ctx, const std::string& array_uri);
+
+TEST_CASE(
+    "SC-60301: Read data with global cell order returns fill values",
+    "[dense-reader][bug][global-cell-order][fixed][sc60301]") {
+  Context ctx;
+  std::string array_uri = "dense_global_cell_order";
```

Comment:
This will use the current working directory right?

Using `VFSTestSetup` is a little better since it will pick a temp dir and put the array in there.
(Admittedly this will be `$(pwd)/tiledb_test` but at least it's easy to see that all the test arrays are in one place)